ui/admin/app/components/form/role/add-principals/index.js

```diff
@@ -35,7 +35,7 @@ export default class FormRoleAddPrincipalsIndexComponent extends Component {
   get filteredPrincipals() {
     // Retrieve principal IDs assigned to current role
     const currentPrincipalIDs = this.args.model.principals.map(
-      (principal) => principal.principal_id,
+      (principal) => principal.id,
```

Comment:
I tested the role/principals workflow to ensure this refactor didn't cause any regression